lix/source-code/client/src/api.test.ts

```diff
@@ -26,7 +26,7 @@ describe("main workflow", () => {
 
 	it("opens a repo url without error and without blocking io", async () => {
 		// fix normalization of .git
-		repository = await openRepository("github.com/inlang/example", {
+		repository = await openRepository("https://github.com/inlang/example", {
```

Comment:
How is a github repository opened without a lix server in between for CORS? 

If the openRepository implicitly routes via a cors proxy, I vote to make that explicit. 

```diff
-repository = await openRepository("https://github.com/inlang/example")
+repository = await openRepository("https://lix.dev/git/github.com/inlang/example")
```

Reply:
@samuelstroschein this would be without any lix server or proxy, just using the lix client eg. in nodejs.   we have to decide if we want to support this usecase in the future when we seperate more from git/github, but for now this is how tests are run and works.